src/components/checkbox/checkbox.tsx

```diff
@@ -29,10 +32,27 @@ const StyledCheckbox = styled(MuiCheckbox, {
   },
 }));
 
-const Checkbox: React.FC<CheckboxProps> = (props) => (
-  <FormGroup>
-    <FormControlLabel control={<StyledCheckbox />} {...props} />
-  </FormGroup>
-);
+const Checkbox: React.FC<CheckboxProps> = (props) => {
+  const {
+    inputLabel,
+    error,
+    errorMessage,
+    ...rest
+  } = props;
+
+  return (
+    <FormGroup>
+      {
+        inputLabel
+        && <InputTitle>{inputLabel}</InputTitle>
+      }
+      <FormControlLabel control={<StyledCheckbox />} {...rest} />
+      {
+        error
+        && <ErrorText sx={{ marginTop: 0 }}>{errorMessage}</ErrorText>
+      }
```

Comment:
I think we should check for `errorMessage` here, too, so as not to render the empty component.

Reply:
That's a good call, I added that check for `errorMessage`